amis/packer_centos7.json

```diff
@@ -228,6 +229,9 @@
           "nvidia" : {
             "enabled" : "{{user `nvidia_enabled`}}"
           },
+          "dcv": {
+            "enabled" : "{{user `dcv_installed`}}"
```

Comment:
why not keep consistency with the variable name?
          "dcv": {
            "installed" : "{{user `dcv_installed`}}"